### PR TITLE
Sisrp 47577 improve display swap class

### PR DESCRIPTION
**Pull request by frankiepun**

### CODE REVIEW

File: src/assets/templates/academics_semester_classes.html [hunk 1]
```diff
@@ -195,21 +195,35 @@ <h3>Swaps</h3>
...
             <tr data-ng-repeat="swapClass in swapClassList">
               <td>
-                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding">&#x21C4;</span>
-              </td>
+                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding" aria-hidden="true">&#x21C4;</span>
+                <span class="cc-visuallyhidden">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-visuallyhidden"> Swap</span> </td>
               <td>
-              <span>
-                <a data-ng-href="{{course.url}}" data-ng-bind="swapClass.swapFromCourse.course_code">
+                <a data-ng-href="{{swapClass.swapFromCourse.url}}" data-ng-bind="swapClass.swapFromCourse.course_code">
```
Comment (marcpechaitis) [inline src/assets/templates/academics_semester_classes.html]: @frankiepun ,
Could this be reduced to
`<span class="cc-visuallyhidden">{{swapClass.swapFromCourse.swap_number}} Swap</span>`

File: src/assets/templates/academics_semester_classes.html [hunk 2]
```diff
@@ -195,21 +195,35 @@ <h3>Swaps</h3>
             <tbody>
             <tr data-ng-repeat="swapClass in swapClassList">
               <td>
-                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding">&#x21C4;</span>
-              </td>
+                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding" aria-hidden="true">&#x21C4;</span>
+                <span class="cc-visuallyhidden">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-visuallyhidden"> Swap</span> </td>
```
Comment (marcpechaitis) [inline src/assets/templates/academics_semester_classes.html]: @frankiepun ,
Could this be reduced to?
`<span class="cc-visuallyhidden">{{swapClass.swapFromCourse.swap_number}} Swap</span>`

**Comment by marcpechaitis** [comment]
@frankiepun could you add a screenshot of how the changes show in CalCentral... I'm just curious.

File: src/assets/templates/academics_semester_classes.html (hunk 2 above)
Comment (frankiepun) [reply src/assets/templates/academics_semester_classes.html]: Excellent feedback. Thanks for pointing it out. I combined the span tags.

**Comment by frankiepun** [comment]
> @frankiepun could you add a screenshot of how the changes show in CalCentral... I'm just curious.

Thanks. Marc. I have attached some screenshots to the ticket. https://jira-secure.berkeley.edu/browse/SISRP-47577. I followed other ARIA example in the code which is supposed to show with the right ARIA tool. Thanks.

File: src/assets/templates/academics_semester_classes.html [hunk 3]
```diff
@@ -195,21 +195,36 @@ <h3>Swaps</h3>
...
+                  </a>
+                  <span data-ng-bind="swapAdditional.title"></span>
+                </span>
               </td>
               <td>
-                <span data-ng-bind="swapClass.dateRequested"></span>
+                <span data-ng-bind="swapClass.swapToSection.last_enrl_dt_stmp"></span>
+                <span data-ng-repeat="swapAdditionalSec in swapClass.swapToAdditionalSection">
+                  <br>
+                  <br>
```
Comment (marcpechaitis) [inline src/assets/templates/academics_semester_classes.html]: @redconfetti, @developish , @nkutub ,
Is it allowed/"normal" for us to use
```
<br>
<br>
```
in this manner?

To be clear I'm not _objecting_, I just want to be sure what he is doing is _right_?

Comment (frankiepun) [reply src/assets/templates/academics_semester_classes.html]: @marcpechaitis  Excellent observation. Actually, I thought it was ok to remove them. Then I found that it was used for handling the second column which may have multiple swap classes with a OR line. So the extra <br>'s ensure that the lines are aligned correctly.  Please refer to the specification attached to the ticket (SISRP-46442 v4.pdf). You will see the "Date Requested" are aligned to handle the multiple courses with OR:

CHEM ENG 24                      August 15, 2019 <br>
OR                                         <br>
CHEM ENG 26                      August 17, 2019

**Pull request merged**